drivers/virtualbox/virtualbox.go

```diff
@@ -182,23 +187,58 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 
 func (d *Driver) PreCreateCheck() error {
 	// Check that VBoxManage exists and works
-	if err := vbm(); err != nil {
-		return err
+	return vbm("--version")
+}
+
+// IsVTXDisabled checks if VT-X is disabled in the bios. If it if, the vm will fail to start at the end.
+// If we can't be sure it is disabled, we carry on and will check the vm logs after we start it.
+func (d *Driver) IsVTXDisabled() bool {
+	if runtime.GOOS != "windows" {
+		// TODO: Check on linux/OSX
+		return false
 	}
 
-	return nil
+	log.Debug("COMMAND: wmic cpu get VirtualizationFirmwareEnabled")
+	stdout, err := exec.Command("wmic", "cpu", "get", "VirtualizationFirmwareEnabled").Output()
+	if err != nil {
+		log.Debugf("Couldn't check that VT-X/AMD-v is enabled. Will check that the vm is properly created.")
+		return false
```

Comment:
But if it's a version of Windows which doesn't have `wmic` for whatever reason, it will return `false` whether or not VT-X/AMD-v is enabled. So it's dangerous, because it could have false negatives.